Sources/RayTracer/RayTracer.Library/Mathematics/MathHelper.cs

```diff
@@ -37,4 +37,89 @@ public static (float, float) SolveQuadraticEquation(float a, float b, float c, o
         rootsCount = 2;
         return (root1, root2);
     }
-}
+
+    public static float[,] CreateRotationMatrix4x4(float xAngle, float yAngle, float zAngle, bool clockwise)
```

Comment:
we should have a dedicated that represents a transform. What about
`
namespace RayTracer.Library.Mathematics;

public readonly struct WorldTransform
{
    public static readonly WorldTransform Identity = // { 1, 0, 0, 0 }, { 0, 1, 0, 0 }, { 0, 0, 1, 0 }, { 0, 0, 0, 1 };
  
    private readonly float[,] _matrix;

    public WorldTransform RotateX(float xAngle, bool clockwise)
    {
        // ...
       return new WorldTransform(rotatedMatrix);
    }

    public WorldTransform RotateY(float yAngle, bool clockwise)
    {
        // ...
       return new WorldTransform(rotatedMatrix);
    }

   // ...
}
`